Services/Interfaces/Models/ArtistLastFm.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace Interfaces.Models
+{
+    public class ArtistLastFm
+    {
+        public string Name { get; set; }
+        public int Playcount { get; set; }
+        public int Listeners { get; set; }
+        public string Mbid { get; set; }
+        public string Url { get; set; }
+        public int Streamable { get; set; }
```

Comment:
You don't need it, please remove it

Reply:
I don't use it, but I need it to get data from API response correctly.
I also can remove ```public List<ImageLastFm> Image { get; set; }```, but will it work correctly?

Reply:
Yes, it will work correctly.
You will need images in the UI story, you can leave them.